src/main/java/org/umcn/me/output/Filter.java

```diff
@@ -0,0 +1,214 @@
+package org.umcn.me.output;
+
+import org.apache.log4j.Logger;
+import org.umcn.me.pairedend.AnchorClusterer;
+import org.umcn.me.pairedend.MobilePrediction;
+import org.umcn.me.util.MobileDefinitions;
+
+import java.io.*;
+import java.util.*;
+
+public class Filter {
```

Comment:
Please rename class to give more context.
Also reconsider if static methods should really be static